three/three.h

```diff
@@ -12,6 +12,21 @@
 #include <Python.h>
 #include <six.h>
 
+#ifndef DATADOG_AGENT_SIX_API
```

Comment:
we should find a way to avoid duplicating around these defines

Reply:
I cleaned that up in #13